Cookbook/Technical-Documents/FeatureSwitches.md

```diff
@@ -119,50 +106,28 @@ All static configuration properties and feature switches should be declared in t
 
-- ### Backend (consumer network/product config) feature switches
+- ### Backend (product config/consumer network) feature switches
 
-	These feature switches are a kind of remote feature switches and are defined on the backend (in Feature Configurator service) and come as a part of patient details and can be accessed as `patient.metadata.featureSwitches`. Typically such feature swithces depend on some user data, i.e. region or consumer network.
+	_Product config_ is a new preferred way of managing such feature switches and it's preferable to use it for any new feature switch that defines features per product/partner. It also aims to replace static configurations and Optimizely (except for its usage for A/B tests and feature experiments) as it can target different applications. 
 	
-	To add a new feature switch on the client add a new property to `BackendFeatureSwitches` struct in `BackendFeatureSwitches.swift` file and code to decode this property in `Decodable.swift`. Of course it needs to be added on the backend as well, that's something the backend dev from your team should be able to help with.
+	To add a new product config feature switch use a `ProductConfigFeatureSwitch` property wrapper instead of `RemoteFeatureSwitch` (as demonstrated later).
 	
-	_Product config_ is a new preferred way of managing such feature switches and it's preferable to use it for any new feature switch that defines features per product/partner. To use product config service in code, use `Current.productConfig` and define the flags via extensions to `ProductConfig` type:
+	Read more about how to define product config flags [here](https://github.com/babylonhealth/manifests/tree/master/product-config#what-is-product-config)
 	
-    ```swift
-    extension ProductConfig {
-        public var myFeature: ProductConfigKeyPath<MyFeautre> {
-            // define key path using a feature key in a product config manifest
-            ProductConfigKeyPath(key: "my_feature")
-        }
-
-        public struct MyFeature: ProductConfigProperty {
-            public static let defaultValue = ProductConfig.MyFeature(
-                myFeatureEnabled: false
-            )
-
-            public let myFeatureEnabled: Bool
-
-            // typical Decodable implementation, value can be represented as a nested object 
-            // or as a single value, then it should be decoded with a `singleValueContainer`
-            private enum CodingKeys: String, CodingKey {
-                case myFeatureEnabled = "my_feature_enabled"
-            }
-        }
-    }
+	Consumer network feature switches are a kind of remote feature switches and are defined on the backend (in Feature Configurator service) and come as a part of patient details and can be accessed as `patient.metadata.featureSwitches`. Typically such feature swithces depend on some user data, i.e. region or consumer network.
```

Comment:
Should we be adding any switches here or use the new product config service for that?